audio.py

```diff
@@ -19,20 +22,46 @@ def read_audio(
 	backend = 'ffmpeg',
 	raw_s16le = None,
 	raw_sample_rate = None,
-	raw_num_channels = None
+	raw_num_channels = None,
 ):
 	try:
 		if audio_path is None or audio_path.endswith('.raw'):
 			if audio_path is not None:
 				with open(audio_path, 'rb') as f:
 					raw_s16le = f.read()
 			sample_rate_, signal = raw_sample_rate, torch.ShortTensor(torch.ShortStorage.from_buffer(raw_s16le, byte_order = byte_order)).reshape(-1, raw_num_channels).t()
-		elif audio_path.endswith('.wav'):
+
+		elif backend == 'scipy' and audio_path.endswith('.wav'):
```

Comment:
Hmm. This would modify the existing behavior of transcribe.py. I'll think more how to do this properly.

Reply:
I think I found a solution (using `backend=None` by default and then using scipy/ffmpeg backends if it's None). I'll push these fixes after we merge this PR